docs/topics/kotlin-notebook-overview.md

```diff
@@ -0,0 +1,92 @@
+
+![Kotlin Notebook](data-analysis-notebook.gif){width=700}
+
+The Kotlin Notebook plugin comes with various features to boost your development process, such as: 
+
+* Accessing APIs within cells
+* Importing and exporting files with a few clicks
+* Using REPL commands for a quick project exploration
+* Getting a rich set of output formats
+* Managing dependencies intuitively with annotations or Gradle-like syntax
+* Importing various libraries with a single line of code or even adding new libraries to your project
+* Getting insights for debugging with error messages and traceback
+
+Kotlin Notebook is based on our [Kotlin Kernel](https://github.com/Kotlin/kotlin-jupyter?tab=readme-ov-file#kotlin-kernel-for-ipythonjupyter), 
+making it easy to integrate with other [Kotlin notebook solutions](https://kotlinlang.org/docs/data-analysis-overview.html#notebooks).
+Without compatibility issues, you can effortlessly share your work among Kotlin Notebook,
+[Datalore](https://datalore.jetbrains.com/), and [Kotlin-Jupyter Notebook](https://github.com/Kotlin/kotlin-jupyter).
+
+With these capabilities, you can embark on a wide range of tasks, from simple code experiments to comprehensive data projects. 
+
+Dive deeper into the sections below to discover what you can achieve with Kotlin Notebook!
+
+## Data analytics and visualization
+
+Whether you're conducting preliminary data exploration or completing an end-to-end data analysis project, Kotlin Notebook has
+the right tools for you.
+
+Within Kotlin Notebook, you can intuitively integrate [libraries](data-analysis-libraries.md) that let you retrieve, transform, plot, and model your data 
+while getting immediate outputs of your operations.
+
+For analytics-related tasks in Kotlin Notebook, the [Kotlin DataFrame](https://kotlin.github.io/dataframe/overview.html) library 
+provides robust solutions. This library facilitates loading, creating, filtering, and cleaning structured data.
+
+Kotlin DataFrame also supports seamless connection with SQL databases and reads data right in the IDE from 
+different file formats, including CSV, JSON, and TXT.
+
+[Kandy](https://kotlin.github.io/kandy/welcome.html), an open-source Kotlin library, allows you to create charts of various types.
+Kandy's idiomatic, readable, and type-safe approach lets you visualize data effectively and gain valuable insights.
+
+![data-analytics-and-visualization](data-analysis-kandy-example.png){width=700}
+
+## Prototyping
+
+Kotlin Notebook provides an interactive environment for running code in small chunks and seeing the results in real-time. 
+This hands-on approach enables rapid experimentation and iteration during the prototyping phase.
+
+With the help of Kotlin Notebook, you can test the concept of solutions early in the ideation stage. Additionally, Kotlin Notebook supports both 
+collaborative and reproducible work, making it easy to ideate.
+
+![kotlin-notebook-prototyping](kotlin-notebook-prototyping.png){width=700}
```

Comment:
I noticed that on many screenshots symbols are grey, it seems you didn't execute the cells to make them resolved (or something is totally broken). Please execute the cells to make these symbols semantically colored, not grey

Reply:
Fixed!